concordium-consensus/src/Concordium/ImportExport.hs

```diff
@@ -454,9 +565,12 @@ exportSections dbDir outDir chunkSize genIndex startHeight blockIndex lastWritte
                                                                     outDir
                                                                     chunkSize
                                                                     lastWrittenChunkM
+                                                                    0
```

Comment:
Why is this correct? What is the meaning of "lastFinalizationIndex" here?

Reply:
Documented this.